charts/dgraph/values.yaml

```diff
@@ -203,9 +203,9 @@ zero:
     successThreshold: 1
 
   readinessProbe:
-    enabled: false
+    enabled: true
     port: 6080
-    path: /health
+    path: /state
```

Comment:
In prod we are using /health endpoint for zero. Should we keep that the same here too?

Reply:
Yeah i think we can keep this to be the same.

Reply:
Edit: Seems like some of the zeros in prod do have the `/state` as the readinessProbe endpoint. So i think this is fine.

Reply:
Yep. Also, the latest cererbo code uses /state endpoint.

`cerebro/region/config/zero/statefulset.json`